docs/SPECS.md

```diff
@@ -395,13 +412,13 @@ derived. The share is encrypted via ECIES using the public key of the recipient
 share holder.  A node bundles all its deals into a `DealBundle` that is
 signed.Here is the protobuf wire specification of the deals:
 
-```protobuf
+```bash
```

Comment:
These ones labelled "protobuf" are not "bash". I would leave these as they were!

Reply:
Again, looks like they're a [recognized syntax](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4171). Rolling back.